web_infrastructure/jboss.py

```diff
@@ -21,7 +21,7 @@
 DOCUMENTATION = """
 module: jboss
 version_added: "1.4"
-short_description: deploy applications to JBoss
+short_description: deploy applications to JBoss 7 EAP
```

Comment:
jboss-cli command, which is explained here only exists since jBoss EAP 6. 
So we could change the short_description to "deploy applications for JBoss 6 EAP and beyond" ?